tests/integration/test_fim/conftest.py

```diff
@@ -55,8 +73,63 @@ def install_audit(get_configuration):
         audit = "auditd"
         option = "--yes"
     else:
-        raise ValueError(f"Linux distro ({linux_distro}) not supported for uninstall/install audit")
+        # Install audit and start the service
+        process = subprocess.run([package_management, "install", audit, option], check=True)
+        process = subprocess.run(["service", "auditd", "start"], check=True)
+
+
+@pytest.fixture(scope='session')
+def configure_local_internal_options_fim():
+    """Fixture to configure the local internal options file."""
+
+    # Backup the old local internal options
+    backup_local_internal_options = get_wazuh_local_internal_options()
+
+    # Set the new local internal options configuration
+    set_wazuh_local_internal_options(create_local_internal_options(FIM_DEFAULT_LOCAL_INTERNAL_OPTIONS))
+
+    yield
+
+    # Backup the old local internal options cofiguration
+    set_wazuh_local_internal_options(backup_local_internal_options)
```

Comment:
Why do you add it there? In the `integration/conftest.py` file are these methods.

I read the `description`, but could you clarify your decision applied here? I do not see sense.

Reply:
This is not the same as the one in `integration/conftest.py`, which needs the `get_local_internal_options` fixture to be present in the test module, and that uses the `get_configuration` fixture.